submissions/promiseChilufya/Database/init.sql

```diff
@@ -0,0 +1,50 @@
+CREATE database food_bag;
+
+USE food_bag;
+/* This is the food table */
+
+	CREATE TABLE food_table(
+	id INT NOT NULL AUTO_INCREMENT,
+	food_type VARCHAR(255) NOT NULL,
+	PRIMARY KEY(id)
+	);
+    /* This is the vendor_food table*/
+    
+    CREATE TABLE vendor_food(
+    id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    food_id INT NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    amount double NOT NULL,
+    PRIMARY KEY(id), FOREIGN KEY(food_id) REFERENCES food_table(id)
+    );
+    /* This is the Admins Table */
+    
+    CREATE TABLE admins_table(
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    type INT NOT NULL,
```

Comment:
The system admins can have a type of 0 and vendors a type of 1 when they are saved for differentiation. This points to an ENUM or Boolean